aws-lambdas/iac/cdk/common/stacks/cron.stack.ts

```diff
@@ -0,0 +1,50 @@
+import * as aws from "@cdktf/provider-aws";
+import { TerraformStack } from "cdktf";
+import { Construct } from "constructs";
+import * as random from "../../.gen/providers/random";
```

Comment:
@jamescrowley321  @saurabhjindal-SF Do we import random like this? I am not a good guy on this so just a question